cmd/ndm_daemonset/probe/usedbyprobe.go

```diff
@@ -129,10 +129,11 @@ func (sp *usedbyProbe) FillBlockDeviceDetails(blockDevice *blockdevice.BlockDevi
 
 		// check for ZFS file system
 		fstype := usedByProbe.BlkidIdentifier.GetOnDiskFileSystem()
-
+		zpool := usedByProbe.BlkidIdentifier.GetOnDiskLabel()
 		if fstype == zfsFileSystemLabel {
 			blockDevice.DevUse.InUse = true
-
+			blockDevice.FSInfo.FileSystem = zfsFileSystemLabel
+			blockDevice.Labels[controller.NDMZpoolName] = zpool
```

Comment:
Is this label needed when the disk is being used by cstor also?

Reply:
Just thinking, it might a good idea to show it. Thinking of the following usecase:

* Installed NDM
* Installed a cstor and used the empty disks for creating cstor pools
* Uinstalled a cstor forcefully from the cluster (without deleting volumes/pools) 
* Reisntalled ndm
* This code can show that zpool (or cstor) was present.

Reply:
It will already show that, but in the annotation , it will mark the disk as using legacy UUID, with the partition table UUID of the old cstor installation